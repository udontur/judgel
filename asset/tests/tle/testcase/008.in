3446 2
17714 99854 94897 53809 27160 12105 98562 85846 97034 47631 62905 47158 7764 3061 52524 97665 4546 68445 64908 59695 81741 73028 55774 92404 31796 75500 86597 38126 71944 98457 16122 14198 23823 57637 33197 29926 68795 62495 38345 42349 31687 85217 48927 34609 79306 44401 10995 95516 88314 37805 59183 75175 92894 46797 35883 68575 14268 54087 31244 19974 1149 2040 24263 36979 17826 81814 95652 20392 15593 90621 15133 31146 74558 71776 47641 91341 23360 67482 20255 45279 37074 48194 12337 71804 42192 61022 23844 29747 28699 98494 25675 56789 15650 64748 82911 40066 88814 72913 86394 68380 11006 19986 18935 14827 59597 93772 55870 90237 98017 80341 10363 178 92472 74341 82617 65122 14388 863 80952 50729 61389 42013 63160 98662 54922 783 18572 64140 17233 90474 91355 95187 33910 24931 99326 16408 22695 2079 73304 21152 37167 86308 90523 40912 67815 28478 2738 81045 47479 83867 38674 69806 62030 71605 54741 86573 30078 84441 72992 98412 93270 52564 17292 27463 55104 67832 62693 70057 32484 78031 41159 8288 54997 56424 50098 38785 19597 58445 23184 18283 67020 96545 76339 51188 60481 23812 16722 5572 8665 74861 79218 25473 80113 58434 31541 63990 43374 94855 50416 92801 6727 60179 51275 48689 3848 82653 72435 71680 39122 2046 9617 6899 13401 31517 72925 83227 73507 73592 19467 5673 42965 29292 34012 63418 27301 80040 23751 31861 84656 58394 7034 43160 81004 17669 46693 46311 28298 90402 29787 49138 65844 7389 98407 57002 51259 23655 76792 45396 30727 12306 54615 37719 35467 30422 62732 55352 86179 29172 82638 35651 94209 82867 60060 70028 88421 7327 30248 97945 60715 14895 31459 39070 60919 3325 57981 16654 37516 38274 5490 25403 53626 29214 71968 81679 54270 17279 70943 15341 83774 43458 30265 20510 54681 71618 79658 79701 2282 91731 42546 93453 64779 76840 44625 23956 16047 12662 93913 8138 41525 44120 15397 74961 39826 18697 51057 38804 5321 17251 62797 3304 75521 7108 81463 54776 97913 85033 87623 56637 34214 88501 58235 75581 26367 94539 66289 32446 76201 65490 45038 3177 64563 63546 44424 10930 68752 74064 78981 33060 11989 43937 13778 21544 18054 78822 56698 92639 96552 42308 95733 84905 2187 59108 36405 61035 73068 49364 95813 48699 46463 16227 25657 75131 76710 74494 55066 11252 21138 91591 43043 61390 2682 14548 67172 35802 46099 67579 52372 52037 31388 96128 22709 12054 11935 98323 24700 91368 99868 12951 62813 12199 40104 76016 80106 14529 24663 99763 64606 87648 2626 50213 55450 91135 46557 99912 16040 10009 88927 44438 74168 91192 2485 99548 74852 13296 40232 68821 58830 86229 76019 83100 89344 15841 34740 53874 51477 40770 4398 90116 58758 84708 30341 47029 53149 30363 3527 7468 87531 36368 6862 39385 56218 58981 79370 2305 81706 8579 72605 90076 29135 61470 66089 43844 96786 14156 47967 3146 20540 79622 32559 49879 38127 90319 81275 7652 36677 84570 52075 98944 38955 5701 45510 44727 15485 53576 40717 16492 29551 58148 94918 91376 43778 85850 17507 23817 31998 35035 92161 24708 91214 28159 28568 69469 51124 49284 19875 52019 35954 96263 16469 22956 89802 56916 26830 26349 25141 67389 95957 18994 94015 14043 50935 49659 45506 380 52913 91717 3332 52286 82684 57008 13820 65486 41625 72586 79739 73546 2270 57501 44168 26936 13976 98968 36900 80152 53704 52024 99863 38721 75249 81368 92371 59837 6752 27758 49684 62779 24585 59821 32449 57630 9209 19462 62468 37205 73908 13115 92875 91392 73726 26530 32279 91804 33205 84507 98778 56768 61024 43345 17087 86174 77286 59064 43065 5768 60877 86268 55109 23157 20162 82553 77335 17550 95812 24460 67444 50330 43870 34024 89500 6286 22665 88341 25746 87675 12721 58411 76780 12105 10896 2296 74508 65591 1607 87899 83719 48178 42015 32117 8887 69662 70952 97392 15349 93694 57579 87444 24681 73914 81970 45549 6681 4488 23183 59091 89198 66955 24966 1188 65431 86095 2985 5445 17888 44074 99182 95551 96465 6862 2511 38444 59124 79000 82963 6519 22601 56698 82170 38821 62196 72464 93933 67644 21693 34697 78576 80661 8828 86634 33469 9750 20571 6085 57755 75464 35775 78196 760 48327 76893 11464 98732 27514 36730 84147 37075 11334 11377 68085 16047 28253 3769 67000 85518 74788 12909 92771 4726 63959 44310 65305 55189 62094 64914 75729 36181 77269 79881 46826 37220 5116 16480 24660 54636 75228 28445 19191 14231 20378 71187 13456 16032 29612 41889 76227 46159 87899 34462 59473 1759 966 65324 37881 20317 34455 28739 50475 31732 76419 94307 26368 97695 43015 6994 13536 86279 62207 35760 98588 51936 17239 42264 8770 25745 54233 80207 79051 50619 27179 46600 33952 37637 45273 41742 16974 27780 64644 31735 13233 44818 75658 57846 71374 88604 5761 38077 19766 54208 82073 7268 81682 10196 94257 153 99307 73247 83081 48725 46737 42260 4705 59757 39911 35844 56635 92941 3677 65286 38521 6825 77225 17227 36396 94540 70029 8915 74189 43059 95218 39378 8120 74321 8589 83821 58097 14172 83380 33170 59670 61194 71975 80146 30724 1819 63102 40170 15266 12401 35301 44548 79650 62038 23479 86482 66113 79293 62883 82807 1355 52236 87231 70881 91778 10754 94168 77549 37094 57618 54322 35451 41096 78902 69992 85404 15695 75867 36922 70711 58864 70883 94662 49537 67778 13112 65819 45538 68620 8021 38044 10096 38817 6172 49222 40690 93966 12369 80938 4152 94172 7321 57190 14136 9190 64124 22716 31782 1188 20850 60252 82891 15866 54595 22109 40439 97046 47061 8151 22557 63781 27306 45949 29343 97390 30121 54612 81238 82398 8245 61624 64951 40747 99484 75437 69716 17634 21365 31522 8638 76564 96032 65004 23498 32157 60013 23135 67746 81521 29392 51381 6664 63486 69643 46607 8436 4205 20464 49240 77882 54287 384 94878 86010 83374 58799 26679 33176 55792 5223 62739 55557 91885 79199 61700 15958 51785 91502 76191 13447 53001 99475 30008 33639 12088 94073 70789 93635 17504 62992 69972 67521 89779 4723 94865 59670 50480 51453 97715 25701 98825 23854 25190 34079 39120 85692 86724 84322 3950 2136 49148 54513 92511 42370 95601 5271 37092 53894 75635 41418 45346 29581 9579 93136 72392 72738 72200 23138 25178 84299 60044 22392 46485 27803 45868 32789 80965 26104 16540 28822 17466 1337 93783 95142 33003 97362 25805 71984 42902 44070 27163 46731 9057 29058 40546 13821 40912 57044 49448 55673 10308 82726 59294 55150 83807 50458 50348 14598 88521 74632 8607 25623 33138 25199 12553 2503 25272 75826 81370 99326 67355 47147 20851 54243 59971 96921 54258 29359 22873 7180 54338 23479 62374 53749 19731 49273 48289 20823 63343 18444 30811 81530 53672 27508 88786 77990 5237 36953 70050 30428 24709 60329 26512 90136 62584 37564 13653 99720 83750 46048 5353 99613 29171 53918 98068 12986 14004 41876 99510 70442 65430 83858 53581 56803 72984 91139 15129 61487 47031 29059 32903 53734 89670 91159 80553 70183 70032 79284 79902 23766 18272 20500 22531 33195 64748 48053 3049 46440 73833 38530 98065 82317 26991 25348 56593 94523 10096 11215 50801 33347 20861 98482 40010 21318 38243 19826 79480 49550 19075 44701 16401 2710 51090 93915 61485 4358 83520 93389 59380 44386 68708 43914 8903 97194 18302 40139 60660 20103 18120 58246 51414 37186 61249 5440 84443 62493 61912 71788 16345 88170 583 39068 86181 82473 20930 9422 35086 9692 8732 16486 45239 97438 32147 80573 30175 59290 44891 56118 35832 83563 88295 36120 14231 56090 14041 32212 17669 36028 2487 13113 5055 71516 63088 22746 9214 28033 47719 83248 17204 41426 38587 22279 80499 51418 38011 66167 47848 84938 64516 60740 70769 34795 7525 84 41274 31094 97826 55505 65590 5954 35337 41392 83243 98318 67061 65060 9364 74854 54432 15726 72220 53636 10368 7736 90844 25214 82011 9698 44567 4301 29489 37245 47407 19206 70389 89235 78790 2695 41888 79510 5618 61522 10230 94250 36417 10457 80054 9443 22482 43089 27554 21335 96759 53968 76662 99237 81598 42508 55287 38269 3475 52869 28140 18516 57246 49306 51565 60498 55970 28118 98356 86611 15171 26547 57318 12812 33519 38082 98333 49070 76142 27069 87391 15888 54663 41662 56995 77643 13977 66669 4409 35075 74192 57134 10898 99203 66695 63836 31391 60621 55784 8605 77913 86202 42746 42030 84379 75140 60908 64421 45132 75384 80558 82380 2912 54772 18946 20357 8216 73475 76747 53127 22065 664 29831 59011 28979 37578 14529 70846 41004 82720 39810 46761 43343 20629 1401 94239 51226 86805 48054 28393 35189 88465 43935 39040 81177 61060 78802 69335 87842 6007 49995 43884 80793 42057 54178 34557 77954 36969 84886 53349 83837 52314 44313 53170 61416 96658 60852 75423 99240 88559 77883 53261 60858 59841 20974 72141 67667 94600 42198 6403 96130 7429 6972 69385 66557 27099 87198 22497 95902 71670 20068 63191 41654 26416 8001 68704 19656 50840 44082 52759 63433 70679 13710 68094 54905 66529 46583 81146 19830 96685 98756 38964 59121 86487 22117 73382 34290 52061 34994 3717 35022 383 71493 5342 53793 60668 14176 82184 98657 84227 53065 91899 20216 70697 18608 13199 58878 3040 88507 97026 31534 62053 19479 21254 83821 62413 65165 25535 3689 77916 24306 99776 49055 14769 63706 288 3303 7048 2186 14827 9935 98818 67909 42531 87416 5513 80613 86868 19173 6617 52234 6691 30479 52754 94034 78450 38494 38339 33471 92749 44088 37281 83874 86533 18746 78168 55558 39351 89851 32355 80313 79029 71315 7389 6804 99490 1100 14701 9241 74598 701 59177 63264 41682 11365 50492 90239 20891 60453 78561 29663 35015 15883 64323 48458 99609 41454 90052 10925 4307 34475 75911 5256 39747 37901 60451 81811 86027 29603 16322 25770 75161 9635 5401 47502 1714 98115 97735 68293 36993 96495 59983 30484 56443 67469 32593 42523 75700 29553 2778 84129 99274 71361 83789 50125 86210 21430 93504 20298 60108 92185 29739 51895 2863 61915 97300 74495 38724 77209 18862 18627 13227 30095 55128 53665 49869 3213 68846 73713 55115 73237 84619 50974 92677 22463 75475 593 86803 16295 37572 8291 5254 17710 67622 69122 60951 36271 15836 24747 41835 19385 73526 83205 5877 33630 79840 1648 53573 98816 39049 30896 22462 57558 68946 76987 79806 853 44840 17092 38723 98110 26746 14572 3023 5344 17301 34188 68118 62984 27431 56513 13295 48784 60416 34132 72201 29121 97074 37299 2225 53634 82016 13179 14985 9351 64574 15237 8008 18382 78598 12098 62764 75795 77133 742 56012 85036 96485 85954 62397 8705 47685 60787 95097 32632 93154 8292 72431 91498 13771 21631 20866 94319 490 18847 77265 32429 74679 41005 7232 64725 99616 47063 77639 59255 21034 81813 24059 36687 62262 91306 11218 893 37618 55977 93557 68329 81644 60794 18961 45998 6834 42655 41350 60333 82472 57633 2339 380 16485 27698 23954 2418 11234 38141 37143 28486 34631 97699 74158 17288 61627 79616 54253 28998 54199 64215 46789 64972 49213 32776 83700 14920 26026 88740 7650 81025 6306 55724 71532 31861 45485 98324 96815 3478 24679 5576 11090 43266 6562 91409 12765 27341 51747 81558 34880 86141 4397 24714 60021 27761 91123 55151 92756 44486 96304 23356 59330 7105 78282 69735 7647 43800 72262 33531 66259 63920 48049 65482 91173 67191 37305 7871 13780 25086 78297 76080 69548 96715 35080 99249 81086 35795 71872 63849 56257 30622 72754 15421 10575 5325 51965 71085 23330 68202 30743 63820 54236 11781 57725 88804 76573 56477 70762 25673 7913 48293 23601 24387 39825 76323 33379 48723 15502 98414 80661 55594 69221 68747 3155 92924 985 87083 39587 20628 62491 22002 14877 98048 28307 44403 96531 14088 43697 16867 83329 58833 77972 63365 34346 45480 34976 75019 96634 23820 13717 13733 89553 30587 70119 83419 71461 17804 3095 56911 22217 7424 31853 14142 50868 89170 26908 8566 96877 55637 7841 65655 97311 51778 21317 2652 47355 91526 81821 77992 26553 12905 7919 73311 29031 15367 38879 91038 35417 31810 78912 41117 25350 80370 94606 45175 67467 90023 84818 98991 47448 88990 34666 73153 37332 24802 84035 54740 51297 36619 44388 23187 22246 74071 40386 11311 70826 45857 49111 93864 81624 82543 48166 6128 65524 66062 58447 73697 95684 18892 37431 20782 41139 47872 34256 7738 49571 51863 84675 9655 47954 53022 9438 34059 71362 25396 90908 12343 67130 15564 56338 55661 68612 53079 20420 54916 54502 623 6505 71089 16976 50582 46981 42281 16511 16235 82819 36978 36770 18012 32923 33805 20530 19034 2939 1963 26894 13447 65891 81734 47997 62867 51719 64545 33501 22834 79071 5696 22769 46868 5802 91370 53913 53435 93544 19930 32859 57498 56450 37899 46461 17278 24265 7077 326 62196 62652 74448 38838 90745 79139 63171 93100 37592 70763 40811 28781 94734 80419 91817 61097 69536 43040 35261 56281 53881 8514 15940 43597 51283 59738 69479 40925 98480 31624 7710 97209 52145 53984 32905 44101 30561 84400 85734 40211 59325 87265 20709 29178 11592 81044 78574 92647 37717 2093 1568 20984 36113 73291 33842 79581 25735 40667 98090 25699 32841 26951 47408 27026 41650 55816 47024 30716 66583 62666 89499 23655 44414 2046 87695 83756 50240 47782 3853 19326 6702 90276 31418 39471 80081 8197 62798 7645 44204 16962 95729 16686 35508 65377 26740 78309 90822 237 63601 37062 52980 30063 94270 56146 42119 44917 63015 12447 34650 56916 76229 47553 75882 17864 65695 52518 25053 77392 42844 70869 78230 52302 86275 72591 14779 52873 41870 80134 77313 54996 62868 26985 92871 26449 78396 83117 5243 6793 15646 34721 86379 54084 78104 70676 26843 56959 13751 22070 37145 46629 53502 64412 26979 63905 75366 50717 13623 72955 21643 29403 65702 6604 8766 73587 47348 74404 42781 70696 90734 71292 99598 38876 747 75014 94671 32811 24389 55059 33184 14284 64497 57490 24863 57499 6308 83980 39984 2795 59946 50474 66914 82206 70406 25419 81448 48865 32058 94643 46233 8938 22243 96290 76171 85510 66179 19048 52102 33957 43081 43629 88607 62388 66752 75125 47474 1207 72190 89638 43793 76351 42223 6511 64724 71049 92309 45154 3031 7006 73238 10899 38365 83706 65194 64729 80059 26675 49289 22622 84120 13541 71462 10587 36264 11971 82851 35495 94575 52840 46527 2055 3847 92467 42875 178 4526 5433 29286 74054 46560 73215 45152 37585 2067 72966 82919 79187 62962 53257 90018 98789 66277 92916 89581 69965 55189 18503 68847 31583 87975 60546 45573 35266 25066 73767 27076 38948 20887 586 79290 94059 77940 29767 19088 21618 51526 2317 95663 84806 33249 80262 1609 11490 57283 63997 18216 60912 88347 6992 56452 58942 61088 20121 57441 10813 95803 23521 39487 58173 7264 72205 84079 67865 46844 67625 29162 70293 77266 93720 51158 52688 9569 31928 19877 28195 80698 10526 68710 61302 40041 605 1506 93094 92183 53135 76690 61874 73825 71695 52509 68072 95838 61387 37561 72908 74726 21546 8003 78547 84598 70708 63931 54744 90798 75584 56783 89029 93768 89626 67535 86948 85547 46248 33503 71814 17067 48940 21930 23782 3381 41405 95650 5039 53742 35104 55075 21294 21845 69416 58389 975 91476 59352 55920 42147 31218 38216 26289 11712 73493 19217 26689 84640 66891 54636 73495 5441 84902 13150 26514 3265 32283 44707 28946 40748 79546 33235 46103 76048 76830 98277 16310 73076 18918 95193 18266 63986 14668 709 3608 35074 76177 59852 71963 41108 44918 11967 34105 98645 49899 72782 6497 16657 24400 93551 42668 48276 77295 82063 52581 78905 40003 25581 42992 27903 66325 69050 67801 2900 61664 89372 35569 23430 15955 67614 95669 3963 13618 94853 32980 49298 96702 94668 13101 90781 71266 58320 83575 62455 79471 82948 74286 20502 79846 81263 97564 69623 60984 34050 13371 79594 2113 23283 11519 74844 13422 2919 74148 3816 17492 42547 20254 20214 83073 12787 81430 14882 80926 78945 80887 89339 61569 71289 40611 45601 18661 93877 45430 66187 53945 41482 38020 89634 59674 32412 89924 60030 76379 9375 33935 86587 64424 44277 8907 7745 63343 69958 52882 51381 81273 49546 22349 69655 13872 4723 38243 9343 3291 39051 21510 2205 21999 37498 59390 30156 63107 90198 68855 60137 86357 25866 56621 85804 61986 30314 53585 1782 95502 12068 18401 70070 93291 53087 22759 76572 36942 75800 30114 65928 40660 45912 8238 61135 92328 87816 95422 42642 80038 2165 73874 36583 19034 34508 56520 66427 244 33554 3027 60102 53153 85649 97674 91666 57934 1154 24236 92638 71946 6635 96161 84357 96342 98872 27042 46969 75723 42179 44339 55249 26921 72226 8713 14148 77483 80424 10718 57068 37465 84357 32779 97907 34113 8923 37949 78571 392 24540 76978 7535 40458 71636 12931 31008 45197 25293 19405 71352 57588 44929 92030 69793 19393 22824 50243 95825 11810 29637 15971 83970 39069 69259 8760 80893 3265 80156 20911 15946 42294 46097 31447 84705 58076 26861 20888 85753 7403 72971 87232 37003 46064 71400 79826 12708 14888 52832 17436 9508 32265 43096 13420 50729 11510 47023 6026 98536 13712 68528 20092 83225 95872 46478 66077 52688 24385 44455 77818 23436 45591 97236 23666 95840 74513 31129 65446 66094 52780 91391 37259 95052 18150 30887 96579 38883 530 33271 76474 10393 8533 44678 83945 10478 43429 31851 61869 21426 83463 88637 17127 67218 36323 56856 51658 63261 89792 94461 42265 42619 35995 51510 33284 49561 95347 2320 20626 59977 79996 51647 59794 79515 69374 2641 85557 74260 79096 72049 11915 86689 97743 7263 16047 23812 68862 8503 83067 34161 29908 25023 87830 21495 90077 17061 96321 47226 76772 85475 33612 43477 33355 72899 31478 99304 39644 76753 64245 88267 26424 63354 12463 16420 34226 50661 61257 85174 94040 5510 26018 2412 38901 686 24925 10035 83683 46282 40999 45883 9639 50525 98104 41859 3882 32650 18625 97269 66381 82098 41277 29348 44632 11535 22808 49642 18597 909 58230 77133 77055 55264 39058 66813 54132 4567 59137 79308 39694 8396 98084 31229 71936 413 88357 77899 65344 38542 30197 46968 63061 97500 83640 10491 65006 45161 87959 54201 95914 72674 76895 18561 33906 10225 69105 93042 2120 1604 4818 77145 22220 13824 33356 97632 19446 94672 35573 45044 51153 75338 61580 15043 63146 28931 76806 10305 17663 17198 14215 20147 65208 29933 62611 91567 26570 81387 28191 54051 76264 57546 37773 84342 60888 38211 81693 21203 90371 39895 59557 65216 93435 61266 65632 79754 19133 79405 68039 47213 34557 7012 91000 35500 39209 83483 9821 63514 56525 42314 16077 96238 37721 10400 75006 80456 54473 62107 85581 12655 23541 34014 95971 10902 42123 12932 46811 37272 10907 83982 82126 38716 93262 97779 52419 57606 16168 30290 55045 49153 48389 62292 6118 7779 60357 66477 39277 3908 71236 36345 68442 89710 20387 28306 6375 2892 8559 45670 58858 1810 56534 86392 48487 63174 34975 86665 96643 98395 75268 86377 91579 22200 15088 38964 5405 60654 30075 72568 60589 55957 60583 36508 33553 95917 32873 73570 7407 16019 95426 54366 99366 86501 82187 86228 59145 75460 82301 57705 24091 27390 3738 79839 14153 99600 41387 10329 41355 41466 86195 97577 77642 82842 34409 89784 92137 80061 40024 26914 49410 56142 48347 714 20441 76040 21249 84259 61489 57799 39203 44245 83481 97788 14856 70218 59904 82518 26124 37862 68158 927 56852 93439 54169 8399 97068 46155 32352 48017 79511 9931 68556 40546 68884 75907 97739 87106 6616 73333 53415 82127 79144 89241 49098 80910 23305 6373 69987 66425 13861 73100 61841 28440 67824 29185 99826 9758 59276 95333 48078 15629 2161 18961 10131 21368 33889 38640 40488 86575 88086 10224 41673 88572 63152 2680 38680 94714 61925 93393 82496 79668 16133 44593 66473 43445 79375 15202 55512 41703 6286 79378 50370 53264 40577 7121 84948 85963 67039 65336 80504 54104 58218 87529 98802 69758 85243 8664 72915 36820 37482 26162 11641 56548 18494 35753 61385 83288 11512 39341 57426 95048 15405 53299 31392 65228 15639 96604 16050 42163 42029 62638 43118 3477 90945 50935 94420 42646 49456 89816 49423 25992 57004 78223 5874 22962 22480 60922 48519 53283 75620 81906 31944 13078 2425 57671 63473 78727 67076 39259 73173 89703 621 63388 13090 22262 24187 50372 6290 33141 37224 16319 89881 54513 22022 82684 48467 14765 40095 85475 89963 34520 76061 15195 48487 96002 3978 75873 64557 14626 99805 39113 78605 84734 88027 69113 57853 88724 61164 14935 25473 12130 42426 13459 26445 96029 68412 90956 76885 88747 88782 72868 14462 10836 46661 30108 67672 47294 40111 6244 74740 10686 89019 22273 36242 23542 82727 82659 6558 6839 25772 81527 16288 21101 19105 77831 326 38275 50716 35083 53699 60094 57754 37750 22214 12897 32215 43230 10027 19587 29370 74907 77059 84263 65634 20396 5622 45872 36358 54848 99025 22003 17699 91276 75582 81744 5198 35574 94481 17174 4242 73381 73536 62824 8333 85105 55874 89229 93857 87034 67056 43507 20233 92721 85535 51466 76813 793 99421 1611 6801 61284 64440 80427 74480 40518 98526 13938 66758 61967 52113 4737 93686 76110 14927 17585 94994 64253 66959 87411 56686 2756 94715 99809 23098 36229 14777 29297 54218 99330 3820 230 63145 50946 20608 54368 32624 98741 32533 77765 38158 26806 2256 69481 62455 59601 60848 29401 18977 85166 81379 85264 86937 43613 44947 83078 8876 21433 8339 58494 21793 68956 82359 33180 86129 78527 84787
68195 19307
