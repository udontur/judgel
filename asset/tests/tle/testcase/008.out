11357527217353
